pages/claim/claim.js

```diff
@@ -0,0 +1,279 @@
+        <Typography variant="span" sx={{fontWeight: 'bold'}}>
+          {connectionDetails.connectedChain}
+        </Typography>      
+      </Typography>
+      <Typography paragraph={true}>
+        You will receive your tokens on address: <br />
+        <Typography variant="span" sx={{fontWeight: 'bold', 
+                                        fontStyle: 'italic',
+                                        fontSize: isMob ? "11px" : "initial"}}>
+          {repRecipient}
+        </Typography>
+      </Typography>
+      {
+        query.status === 'init' ?
+          <div>
+            <Box
+            component="form"
+            onSubmit={changeRecipient}
+            sx={{mt: 1}}
+            >
+            <Typography paragraph={true}>
+              If you want to receive your GOOD tokens on a different address, 
+              set a new recipient below.
+            </Typography>
+            <Typography variant="span" color="red">
+              REMEMBER TO ONLY USE ADDRESSESS WHICH ARE YOUR OWN AND SUPPORT ERC-20's
+            </Typography>
+            <div style={{
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center'
+            }}>
+              <TextField
+                error={!isEth.test(newRecValue)}
+                helperText={!isEth.test(newRecValue) ? "This is either not a ETH address, or it doesn't exist" : ''}
+                margin="normal"
+                required
+                id="newRecipient"
+                label="New Recipient"
+                name="newRecipient"
+                onChange={(e) => setNewRecValue(e.target.value)}
+                sx={{mr: 1}} />
+              <Button 
+                type="submit"
+                variant="contained"
+                sx={{
+                  fontSize: '13px', 
+                  mt: 3, 
+                  mb: 2,
+                  backgroundColor: "#00C3AE", 
+                  '&:hover': {
+                    backgroundColor: "#049484"
+                  }
+                }}
+                >Set New Recipient</Button>
+
+            </div>
+          </Box>
+          <Button
+              variant="contained"
+              sx={{ mt: 3, mb: 2}}
+              onClick={() => skipAndClaim()}
+            >Skip</Button>
+          </div>
+        :
+        query.status === 'pending' ?
+          <div style={{
+            display: "flex",
+            alignItems: "center",
+            flexDirection: "column",
+          }}>
+            <CircularProgress color="secondary" sx={{marginTop:"20px"}} /> <br />
+            <Typography variant="span" sx={{fontStyle: 'italic'}} color="red">
+                You have a current pending transaction, please wait till confirmation.
+            </Typography>
+          </div> 
+        :
+        query.status === 'claim-start' || 
+        query.status === 'claim-success' || 
+        query.status === 'claim-failed' ?
```

Comment:
can this be simplified to query.status!=='claim-init'?

Reply:
yes it can!